prescription.sh

```diff
@@ -183,6 +187,10 @@ function generateYML () {
 	    s~<<JIRA_ISSUES_QUERY>>~$jira_issues_query~g; \
 	    s~<<JIRA_USERNAME>>~$jira_username~g; \
 	    s~<<JIRA_AUTH_TOKEN>>~$jira_auth_token~g; \
+            s~<<RALLY_PROJECT_NAME>>~$rally_project_name~g; \
```

Comment:
indent needs to be same as the above code

Reply:
indentation looks good on the branch -  here it shows that way
https://github.com/synopsys-sig/io-artifacts/blob/POLCT-157/prescription.sh

Reply:
Not sure might why it shows that way, but anyways thanks